docs/get-started/sveltekit.md

```diff
@@ -0,0 +1,350 @@
+| [`$service-worker`](https://kit.svelte.dev/docs/modules#$service-worker)           | ⛔ Not supported       | This is a service worker feature, which does not apply to Storybook.                                                                    |
+
+## How to mock
+
+To mock a SvelteKit import you can define it within `parameters.sveltekit_experimental`:
+
+```ts
+// MyComponent.stories.svelte
+export const MyStory = {
+  parameters: {
+    sveltekit_experimental: {
+      stores: {
+        page: {
+          data: {
+            test: 'passed',
+          },
+        },
+        navigating: {
+          route: {
+            id: '/storybook',
+          },
+        },
+        updated: true,
+      },
+    },
+  },
+};
+```
+
+The [available parameters](#parameters) are documented in the API section, below.
+
+### Mocking links
+
+The default link-handling behavior (e.g. when clicking an `<a href="..." />` element) is to log an action to the [Actions panel](../essentials/actions.md).
+
+You can override this by assigning an object to `parameters.sveltekit_experimental.hrefs`, where the keys are strings representing an href and the values define your mock`. For example:
+
+```ts
+// MyComponent.stories.svelte
+export const MyStory = {
+  parameters: {
+    sveltekit_experimental: {
+      hrefs: {
+        '/basic-href': (to, event) => {
+          console.log(to, event);
+        },
+        '/root.*': {
+          callback: (to, event) => {
+            console.log(to, event);
+          },
+          asRegex: true,
+        },
+      },
+    },
+  },
+};
+```
+
+## Writing native Svelte stories
+
+Storybook provides a Svelte addon maintained by the community, enabling you to write stories for your Svelte components using the template syntax. You'll need to take some additional steps to enable this feature.
+
+Run the following command to install the addon.
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+   'svelte/svelte-csf-addon-install.yarn.js.mdx',
+   'svelte/svelte-csf-addon-install.npm.js.mdx',
+   'svelte/svelte-csf-addon-install.pnpm.js.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+<Callout variant="info">
+
+The community actively maintains the Svelte CSF addon but still lacks some features currently available in the official Storybook Svelte framework support. For more information, see [addon's documentation](https://github.com/storybookjs/addon-svelte-csf).
+
+</Callout>
+
+## API
+
+### Parameters
+
+This framework contributes the following [parameters](../writing-stories/parameters.md) to Storybook, under the `sveltekit_experimental` namespace:
+
+#### `forms`
+
+Type: `{ enhance: TK }`
+
+Provides mocks for the `$app/forms` module.
+
+##### `forms.enhance`
+
+Type: TK
```

Comment:
What is type TK in this whole section?

Reply:
Thanks for all the suggestions! I opted to handle them in a single commit along with some other updates, so it appears I did not accept them. But I did!